src/components/Proposal/Create/CreateProposalPage.tsx

```diff
@@ -62,8 +62,13 @@ class CreateProposalPage extends React.Component<IProps, IStateProps> {
     history.push("/dao/" + daoAvatarAddress + "/plugin/" + pluginId);
   }
 
+  private preventDefault = (e: any) => {
+    e.preventDefault();
+  }
+
   public async componentDidMount() {
     document.addEventListener("keydown", this.handleKeyPress, false);
+    (document.getElementsByTagName("BODY")[0] as HTMLElement).style.overflow = "hidden";
```

Comment:
I thought about this but for some wired reason `ModalPopup` is not the modal that is used in the modal of the `createProposalPage`, hence changing the `.tsx/.scss` in `ModalPopup` is not affecting `createProposalPage modal`. That's another bad practice that the "generic" modal is not used all across Alchemy.

Reply:
Ah yes, I forgot about where we use `ModalRoute`.  I believe the purpose is that the popup be accessible via defined routes that match a browser URL.

All you have to do is add this to the :global section at the top of CreateProposal.scss

```
  body.react-router-modal__modal-open {
    overflow: hidden;
  }
```

I totally agree that this is not ideal structure.  We might be able to refactor the structure considering the ModalRoute properties here:  https://github.com/davidmfoley/react-router-modal#modalroute, but I believe we should do that in a separate PR since it involves very different use cases for testing.

Reply:
We should also look at uses of SimpleMessagePopup, though I don't know if that is in 2.0 yet.